ols/app/main.py

```diff
@@ -43,8 +43,8 @@
 
 # update provider and model as soon as possible so the metrics will be visible
 # even for first scraping
-metrics.selected_provider.info({"name": config.ols_config.default_provider})
-metrics.selected_model.info({"name": config.ols_config.default_model})
+metrics.selected_provider.info({"name": config.ols_config.default_provider or ""})
+metrics.selected_model.info({"name": config.ols_config.default_model or ""})
```

Comment:
this doesn't seem aligned w/ the PR title.

if the change still makes sense i guess you can pull it into your new metrics PR that's touching this space